src/crimson/os/seastore/onode_manager/staged-fltree/fwd.h

```diff
@@ -90,4 +93,79 @@ inline MatchKindCMP reverse(MatchKindCMP cmp) {
   }
 }
 
+enum class node_delta_op_t : uint8_t {
```

Comment:
hmm. this is not a "fwd" declaration anymore. i'd suggest move this type definition to somewhere else.

Reply:
moved to `node_types.h`